prov/hook/trace/src/hook_trace.c

```diff
@@ -150,6 +150,36 @@ trace_cq(struct hook_cq *cq, const char *func, int line,
 	}
 }
 
+static inline void
+trace_cq_err(struct hook_cq *cq, const char *func, int line,
+             struct fi_cq_err_entry *entry,  uint64_t flags)
+{
+	char err_buf[80];
+
+	if (!fi_log_enabled(cq->domain->fabric->hprov, FI_LOG_TRACE, FI_LOG_CQ)) {
+		return;
+	}
```

Comment:
nit: braces aren't needed.  (I wouldn't have bothered with this comment, only a separate update is needed.)